src/common/snippets/src/utils.cpp

```diff
@@ -132,6 +133,14 @@ VectorDims get_planar_vdims(const snippets::lowered::ExpressionPort& expr_port)
     return get_planar_vdims(expr_port.get_descriptor_ptr());
 }
 
+bool is_dynamic_vdims(const VectorDims& shape) {
+    for (const auto& v : shape) {
+        if (v == IShapeInferSnippets::DYNAMIC_DIMENSION)
+            return true;
+    }
+    return false;
```

Comment:
As you remember, I like stl functions 😃 
```suggestion
    return std::any_of(shape.cbegin(), shape.cend(), [](const size_t& v){ return v == IShapeInferSnippets::DYNAMIC_DIMENSION; });
```

Reply:
Ok, even clang is on your side with this one)